docs/source/using/addons.adoc

```diff
@@ -76,6 +78,32 @@ If the metadata field *OpenShift-Version* is not specified in the add-on header,
 *OpenShift-Version* only supports versions in the form of <major>.<minor>.<patch>.
 ====
 
+[[addon-minishift-version-semantics]]
+== Minishift-Version Semantics
+
+As part of the add-on metadata you can specify the {project} version which needs to be running in order to apply the add-on.
+To do so, you can specify the optional *{project}-Version* metadata field.
```

Comment:
I am not sure if `{project}` should be used here. It is not like this becomes `CDK-Version`, right?